test/pages/InternationalAddressPageSpec.scala

```diff
@@ -31,7 +32,9 @@ class InternationalAddressPageSpec extends AnyFreeSpec with Matchers {
 
       "must go to Index" in {
 
-        InternationalAddressPage.nextPage(NormalMode, emptyAnswers) mustEqual routes.IndexController.onPageLoad()
+        val answers = emptyAnswers.set(InternationalAddressPage, InternationalAddress("Testing Lane", Some("Testing Road"), "New York", None, Some("AB1 A"),
```

Comment:
This is fine, but just for info - we only need to set answers in these tests when the navigation depends on it.  So for example, on a yes/no page the actual navigation might depend on the answer, so we need to set it in the tests.  Any pages where we're always going to the same place don't need to have anything set.